lib/database/adapters/RunAdapter.js

```diff
@@ -125,6 +125,8 @@ class RunAdapter {
             logs,
             definition,
             calibrationStatus,
+            userIdO2Start,
+            userIdO2Stop,
```

Comment:
Could you just put these after timeTrgEnd? Same for everywhere they all appear

Reply:
done